addons/l10n_ph/data/account_tax_template_cash.xml

```diff
@@ -0,0 +1,2803 @@
+<?xml version='1.0' encoding='utf-8' ?>
```

Comment:
what's that file for? 

is that all duplicated taxes meant to be cash-based (but with cash basis unset for any reason?) ??